util/design/data/prim_mubi_pkg.sv.tpl

```diff
@@ -12,23 +12,21 @@
 
 package prim_mubi_pkg;
 
-% for n in range(1, n_max_nibbles + 1):
 <%
-  nbits = n * 4
-  true_val = ''
-  false_val = ''
-  for k in range(1,n+1):
-    true_val = ('A' if (k % 2) else '5') + true_val
-    false_val = ('5' if (k % 2) else 'A') + false_val
+import prim_mubi as prim_mubi
+%>\
+% for n in range(1, 4 + 1):
```

Comment:
should we replace the literal `4` with the global pythan variable `N_MAX_NIBBLES` from `prim_mubi.py`?